src/common/rbac/const.go

```diff
@@ -55,6 +55,7 @@ const (
 	ResourceScanner               = Resource("scanner")
 	ResourceArtifact              = Resource("artifact")
 	ResourceTag                   = Resource("tag")
+	ResourceAccessory             = Resource("accessory")
```

Comment:
We need to update the permissions map of the role.

Reply:
added, thanks.